### DISCUSSION TITLE
CMD+? Comments code instead of showing the OpenAI Assistant window

### DISCUSSION THREAD

**Discussion by MD5-Hashm** [zed-industries/zed#12668] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

Hovering over the Assistant button provides the alt text of "CMD+?" however cmd+? comments how your current line of code.

### Environment

OSX 14.2.1

**Comment by SomeoneToIgnore** [comment]
Do you use the En-US layout that has / and ? on the same key?
If so, did you really press `cmd-shift-/` (that is `cmd-?`) instead of `cmd-/`?

**Comment by MD5-Hashm** [comment]
I see my issue
